pages/glossary/different-hue.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Different hue
+key: different-hue
+unambiguous: true
+objective: true
+---
+
+Two sets of colors have _different hue_ if the hue of the colors in one set is different from the hue of the colors in the other set.
```

Comment:
When comparing sets, do you mean "pairwise different" (no element in common) or "globally different" (at least one element different)?